src/Data/List/Relation/Binary/Lex/Strict.agda

```diff
@@ -45,11 +45,11 @@ module _ {a ℓ₁ ℓ₂} {A : Set a} where
       _≋_ = Pointwise _≈_
       _<_ = Lex-< _≈_ _≺_
 
-    xs≮[] : ∀ xs → ¬ xs < []
-    xs≮[] _ (base ())
+    xs≮[] : ∀ {xs} → ¬ xs < []
+    xs≮[] (base ())
```

Comment:
Was this added in `v2.0`? If not, it probably needs a CHANGELOG entry?

Reply:
So, I added the `List` version in #1648 and the `Vec` version in #1672 each of which made it into 1.7*, I think, so yes, I suppose so.